docs/content/documentation/systems/AuxKernels/tensor_mechanics/VectorRankFourAux.md

```diff
@@ -0,0 +1,10 @@
+<!-- MOOSE Documentation Stub: Remove this when content is added. -->
```

Comment:
We need to have more than just stubs for these.  We're requiring that all new code have documentation files with them now.

Reply:
I have to admit this was mostly to make the Documentation test pass... I will provide more details soon!